regolith/schemas.py

```diff
@@ -3252,6 +3252,7 @@
                 "poster",
                 "seminar",
                 "tutorial",
+                "webinar"
```

Comment:
since we are touching this code now, I want to change how we do these eallowed by defining a global variable at the top of the schemas file and then insert it here.  the advantage of this is that we can import this into helpers to get the allowed lists that are synchronized.   We could call it by default ``<COLLECTION>_<FIELD1>_<FIELD2>`` etc., so this one would be called ``PRESENTATIONS_TYPE`` and it would be set to this list at the top of the file.

Reply:
done!